Gemfile.lock

```diff
@@ -841,9 +845,11 @@ GEM
       equalizer (~> 0.0, >= 0.0.9)
     virtus-multiparams (0.1.1)
       virtus (~> 1.0)
-    voting_schemes-dummy (0.15.2)
+    voting_schemes-dummy (0.16.0)
+      mimemagic (~> 0.3.7)
```

Comment:
Actually, before merging this: do you need this gem in your dependency? Could you try replacing this one by https://github.com/rails/marcel ? `mimemagic` requires users to download a file manually to work, apparently...

Reply:
That is: I'd try to avoid adding the `mimemagic` dependency to Decidim to avoid problems. We have already updated all gems that depended on it, they all seem to use `mrcel` now.